test/CMakeLists.txt

```diff
@@ -43,3 +43,32 @@ compile_benchmark_test(cxx03_test)
 set_target_properties(cxx03_test
     PROPERTIES COMPILE_FLAGS "${CXX03_FLAGS}")
 add_test(cxx03 cxx03_test --benchmark_min_time=0.01)
+
+# Add the coverage command(s)
+if (${CMAKE_BUILD_TYPE} MATCHES "[Cc]overage")
+  find_program(GCOV gcov)
+  find_program(LCOV lcov)
+  find_program(GENHTML genhtml)
+  # TODO: support more compilers and generators
+  find_program(MAKE make)
+  if (GCOV AND LCOV AND GENHTML AND MAKE AND "${CMAKE_GENERATOR}" STREQUAL "Unix Makefiles")
+    add_custom_command(
+      OUTPUT ${CMAKE_BINARY_DIR}/lcov/index.html
+      COMMAND ${LCOV} -q -z -d .
+      COMMAND ${LCOV} --no-external -q -c -b "${CMAKE_BINARY_DIR}" -d . -o initial.lcov  -i
```

Comment:
It depends on the source being built _with_ coverage. In the `CMakeList.txt` file in the route it sets up the `COVERAGE` build variant. It adds the `--coverage` flag.